angular/projects/admin/src/app/admin/document/document-list/document-list.component.ts

```diff
@@ -62,8 +68,8 @@ export class DocumentListComponent implements OnInit, OnDestroy, AfterViewInit {
     const url = `/_/admin/document/${documentId}`;
     this.router.navigateByUrl(url);
   }
-  setTotalCount (dataSource: Object) {
-    const refNumEntries = dataSource['documentType'].documentCount;
+  setTotalCount(dataSource: Object) {
+    const refNumEntries = dataSource['documentType'].numEntries;
```

Comment:
Seems like the previous use of `documentCount` is should be used. Perhaps it's just the database that's having the wrong data.

Reply:
Resolve on this commit https://github.com/oddbit/tanam/pull/193/commits/50bb2072b77606029542acdf6e0c385b91328c33